app/views/users/index.html.haml

```diff
@@ -1,16 +1,15 @@
-%h1 Users
+%h3 All Users
 
 %table
   %thead
     %tr
       %th
-        Email
-      %th
+        Name
       %th
+        Email
 
   %tbody
     - @users.each do |user|
       %tr
+        %td= link_to user.name, profile_path(user)
```

Comment:
- Use one of the helper methods. 
- Profile path policy is limited to current_user. Add an edit and update action for user instead.